doc/source/guide/geometry_guide.rst

```diff
@@ -0,0 +1,286 @@
+
+        R^{(\mathrm{i})} v &= \cos\beta v + \sin\beta (a \times v) + (1 - \cos\beta)\langle a,\, v \rangle a.
+
+This construction becomes unstable when :math:`v_1^{(\text{def})} \approx v_1`.
+Therefore, if :math:`\big\| v_1^{(\text{def})} - v_1 \big\| < \delta` for a threshold :math:`\delta`, we take :math:`v_1 = v_1^{(\text{def})}`.
+
+Below are two concrete examples for how this computation is done in practice.
+For a list of implemented geometries, check the API documentation of the ``odl.tomo.geometry`` subpackage.
+
+`Parallel3dEulerGeometry`
+^^^^^^^^^^^^^^^^^^^^^^^^^
+In this geometry, the motion parameters are two or three Euler angles, and the detector is two-dimensional and flat.
+The handle :math:`v_1` for the initial rotation is the initial detector position, provided as ``det_pos_init`` parameter.
+Its default value is :math:`v_1^{(\text{def})} = (0, 1, 0)^{\mathrm{T}}`.
+
+`Parallel3dAxisGeometry`
+^^^^^^^^^^^^^^^^^^^^^^^^
+This geometry, like all subclasses of `AxisOrientedGeometry`, has a fixed symmetry axis, provided as ``axis`` parameter.
+Its motion parameter is a single rotation angle around this axis.
+The initial orientation handle :math:`v_1` is the symmetry axis, with default value :math:`v_1^{(\text{def})} = (0, 0, 1)^{\mathrm{T}}`.
+
+`CircularConeFlatGeometry`
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+The 3D cone beam geometry with circular acquisition curve is also an `AxisOrientedGeometry`.
+Here, the symmetry ``axis`` is perpendicular to the source and detector circles (which can be different but lie in the same plane).
+Its motion parameter is a single angle that parametrizes the position of the source on the circle, and the detector lies opposite of the source point, at maximum distance.
+As in `Parallel3dAxisGeometry`, the initial orientation is determined by the symmetry axis, with the same default.
+
+.. figure:: figures/circular_cone3d_sketch.svg
+    :width: 75%
+    :align: center
+
+
+Detector properties
+-------------------
+The detector model in ODL is intended to be very flexible and able to encode many different types of detectors.
+Besides the obvious flat 2d detectors, it is also possible to implement curved detectors as used in medical CT, PET detector rings, Compton cameras, point-like transducers etc.
+
+Nevertheless, names and concepts are centered around the surface-like detector model since it is most widely used in practice.
+In particular, the function :math:`s(u)` mapping a detector parameter to a point on the detector (e.g. two angles to a point on a curved detector surface) is called ``surface``.
+
+There are two methods that can be implemented for additional functionality, ``surface_deriv`` and ``surface_measure``.
+The former should be the derivative map
+
+    .. math::
+        & \partial s : D \to TM_{\mathrm{d}},
+
+        & (\partial s)_i = \frac{\partial s}{\partial u_i}
+
+from :math:`D` to the tangent bundle :math:`TM_{\mathrm{d}}` of the detector manifold :math:`M_{\mathrm{d}}`.
```

Comment:
Not quite sure if this is the correct level of abstraction here, but keep it as is for now. We could use some simplified examples at some points for non math PhDs.